apps/src/code-studio/components/playzone.story.jsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import PlayZone from './playzone';
-import CreateSomething from './lessonExtras/CreateSomething';
 
-export default storybook => {
-  storybook.storiesOf('PlayZone', module).addStoryTable([
-    {
-      name: 'Default',
-      story: () => <PlayZone lessonName="Test Lesson" onContinue={() => {}} />
-    },
-    {
-      name: 'Create something',
```

Comment:
I did not refactor this story because CreateSomething is a component in Playzone, so it's already visible in the Playzone story.